.github/workflows/build.yaml

```diff
@@ -45,7 +45,6 @@ jobs:
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-build.yaml@branch-25.08
     with:
-      matrix_filter: map(select((.CUDA_VER | startswith("12"))))
```

Comment:
Recognize we are only testing CUDA 12, but could we keep this?

Reply:
Since we might move to cuda 13 in 25.10 or later, I removed it just for future case.